src/Verbs/PushCommand.cs

```diff
@@ -72,17 +78,64 @@ public static async Task<int> RunAndReturn(PushOptions options, PackageHelper pa
                 packageHelper.EnsurePackageDoesntAlreadyExists(packages, filePath);
             }
 
+            // Archive packages
+            var archivePatterns = new List<string>();
+            var packagesToArchive = new List<int>();
+
+            if (options.Archive != null)
+            {
+                archivePatterns.AddRange(options.Archive);
+            }
+
+            // TODO: Once a "Current" option is introduced, which makes the current package being upload "Current"
+            // then uncomment the following lines
+            //if (options.Current && !archivePatterns.Contains("current"))
+            //{
+            //    archivePatterns.Add("current");
+            //}
+
+            if (archivePatterns.Count > 0)
+            {
+                foreach (var archivePattern in archivePatterns)
+                {
+                    if (archivePattern == "current")
+                    {
+                        // If the archive option is "current", then archive the current package
+                        var currentPackage = packages.FirstOrDefault(x => x.Value<bool>("Current"));
```

Comment:
Hmm, that kind of depends what the default behavior is no archive flags are supplied? 

I think this would ultimately mean I still couldn't use it for Vendr (nor could you for Umbraco Forms) because where you have a project listing multiple package files, and the secondary packages essentially contain the primary packages name in them, there is no wildcard that could exclude those.

For example, my packages are `Vendr.1.2.3.zip` for the core package with payment provider packages like `Vendr.PaymentProvider.Stripe.1.2.3.zip`. For the payment providers it's cool, because I can use an archive wildcard like `Vendr.PaymentProviders.Stripe.*.zip` but I can't use a wildcard for the core package, as it would be `Vendr.*.zip` and so this would match all the packages listed and archive everything.

So sure, it could be the initial version, but until this is resolved, I can't use it myself.

Reply:
Well, I guess you could do an ugly workaround for the time being and for the core package you have an archive flag like `-a Vendr.1.*.zip Vendr.2.*.zip Vendr.3.*.zip` 🤔